src/styles/main.scss

```
@@ -1,3 +1,50 @@
+$minSize: 300px;
+
+$backgroundColor:
+  1 rgb(251, 0, 0),
+  2 rgb(200.8, 0, 0),
+  3 rgb(150.6, 0, 0),
+  4 rgb(100.4, 0, 0),
+  5 rgb(50.2, 0, 0),
+  6 rgb(0, 0, 0);
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
```

if you already create some vars why you don't create for colors?)

```suggestion
  color: $c-white;
```